### PR TITLE
added support for powerpoint files

### PR DESCRIPTION
**Pull request by sam0044**
### Contribution Checklist

- [ ] The code builds clean without any errors or warnings
- [ ] The PR follows the [Contribution Guidelines](https://github.com/microsoft/chat-copilot/blob/main/CONTRIBUTING.md) and the [pre-submission formatting script](https://github.com/microsoft/chat-copilot/blob/main/CONTRIBUTING.md#development-scripts) raises no violations
- [ ] All unit tests pass, and I have added new tests where possible
- [ ] I didn't break anyone :smile:

### CODE REVIEW

File: webapi/Services/DocumentTypeProvider.cs [hunk 1]
```diff
@@ -33,6 +33,8 @@ public DocumentTypeProvider(bool allowImageOcr)
                 { FileExtensions.ImageJpg, true },
                 { FileExtensions.ImageJpeg, true },
                 { FileExtensions.ImageTiff, true },
+                { ".ppt", false },
```
Comment (wdavidcamden) [inline webapi/Services/DocumentTypeProvider.cs:36]: can we use the FileExtensions class for the two file extensions?

Comment (sam0044) [reply webapi/Services/DocumentTypeProvider.cs:36]: The FileExtensions class is read only. I tried overwriting it but changes to it don't show up in git for some reason. I don't see it in git ignore either

Comment (wdavidcamden) [reply webapi/Services/DocumentTypeProvider.cs:36]: it has a an extension for ppt and pptx. I think its FileExtensions,MsPowerPoint and FileExtensions.MsPowerPointX

Comment (sam0044) [reply webapi/Services/DocumentTypeProvider.cs:36]: oh i see it. I am blind nvm

**Review by wdavidcamden — APPROVED** [review]

**Pull request merged**